spec/index.bs

```diff
@@ -1762,12 +1764,12 @@ enum GPUAddressMode {
 
     : <dfn>"mirror-repeat"</dfn>
     ::
-        Texture coordinates between -1.0 and 1.0, inclusive, are mirrored across the axis.
-        Outside -1.0 and 1.0, the texture coordinates wrap to the other side of the texture.
+        Texture coordinates wrap to the other side of the texture, but the texture is flipped
+        when the integer part of the coordinate is odd. 
 </dl>
 
-{{GPUFilterMode}} describes the behavior of the sampler if the provided texture coordinates do not
-correspond to a single texel.
+{{GPUFilterMode}} describes the behavior of the sampler if the sample footprint is smaller or larger
```

Comment:
that's not correct. The footprint can be exactly one texel (note: please use "texel" consistently), and we still need to filter it.
What we should be saying here is that "sample footprint does not exactly match a texel". This sounds similar to the previous statements, but there is a critical hidden difference: we are talking about the **shape** of the footprint here, where other places only talk about the **area** of the footprint. I think we need to change the other places to say "footprint area" when comparing to 1 everywhere, for clarity.

Reply:
Hmm, I thought I replaced all mentions of "pixel" with "texel"; ok